app/src/main/res/xml/xdrip_plus_prefs.xml

```diff
@@ -36,30 +36,30 @@
                         android:title="@string/high_glucose_values"
                         app:colorpicker_noneSelectedSummaryText="@string/default_color_selected"
                         app:colorpicker_selectNoneButtonText="@string/revert_to_default"
-                        app:colorpicker_showHex="false" />
+                        app:colorpicker_showHex="true" />
```

Comment:
The default value is `true`. Hence, this attribute can be removed completely, also in the following preferences.